ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -64,14 +64,14 @@ pub(crate) struct Module {
     consts: TiVec<ConstIdx, Constant>,
     /// The type table.
     ///
-    /// A [TypeIdx] describes an index into this.
-    types: TiVec<TypeIdx, Type>,
+    /// A [TyIdx] describes an index into this.
```

Comment:
Commit message is inaccurate? Should it read "TypeIdx to **TyIdx**"?

Reply:
Oops, you're right, https://github.com/ykjit/yk/pull/1122/commits/668ac40acb8075cddea3f72bb3316bd0e44210f9 is incorrectly titled.